data/tools/red_teaming.json

```diff
@@ -63,6 +63,22 @@
       "language": "Shell",
       "price": "Free",
       "online": "False"
-    }
+    },
+    {
+      "name": "King Phisher",
```

Comment:
You have to sort them alphabetically. see [Format conventions](https://inventory.raw.pm/docs/docs/en/format/) & [Adding/changing content](https://inventory.raw.pm/docs/docs/en/add/#addingchanging-content)